test/test_nimiq_client.py

```diff
@@ -0,0 +1,933 @@
+        self.assertEqual(PeerAddressState.ESTABLISHED, result.addressState)
+        self.assertEqual(PeerConnectionState.ESTABLISHED, result.connectionState)
+
+    def test_sendRawTransaction(self):
+        SessionStub.testData = Fixtures.sendTransaction()
+
+        result = self.client.sendRawTransaction("00c3c0d1af80b84c3b3de4e3d79d5c8cc950e044098c969953d68bf9cee68d7b53305dbaac7514a06dae935e40d599caf1bd8a243c00000000000000010000000000000001000dc2e201b5a1755aec80aa4227d5afc6b0de0fcfede8541f31b3c07b9a85449ea9926c1c958628d85a2b481556034ab3d67ff7de28772520813c84aaaf8108f6297c580c")
+
+        self.assertEqual("sendRawTransaction", SessionStub.latestRequestMethod)
+        self.assertEqual("00c3c0d1af80b84c3b3de4e3d79d5c8cc950e044098c969953d68bf9cee68d7b53305dbaac7514a06dae935e40d599caf1bd8a243c00000000000000010000000000000001000dc2e201b5a1755aec80aa4227d5afc6b0de0fcfede8541f31b3c07b9a85449ea9926c1c958628d85a2b481556034ab3d67ff7de28772520813c84aaaf8108f6297c580c", SessionStub.latestRequestParams[0])
+
+        self.assertEqual("81cf3f07b6b0646bb16833d57cda801ad5957e264b64705edeef6191fea0ad63", result)
+
+    def test_createRawTransaction(self):
+        SessionStub.testData = Fixtures.createRawTransactionBasic()
+
+        transaction = {
+            "from": "NQ39 NY67 X0F0 UTQE 0YER 4JEU B67L UPP8 G0FM",
+            "fromType": AccountType.BASIC,
+            "to": "NQ16 61ET MB3M 2JG6 TBLK BR0D B6EA X6XQ L91U",
+            "toType": AccountType.BASIC,
+            "value": 100000,
+            "fee": 1,
+            "data": None
+        }
+
+        result = self.client.createRawTransaction(transaction)
+
+        self.assertEqual("createRawTransaction", SessionStub.latestRequestMethod)
+
+        param = SessionStub.latestRequestParams[0]
+        self.assertEqual(param, {
+            "from": "NQ39 NY67 X0F0 UTQE 0YER 4JEU B67L UPP8 G0FM",
+            "fromType": 0,
+            "to": "NQ16 61ET MB3M 2JG6 TBLK BR0D B6EA X6XQ L91U",
+            "toType": 0,
+            "value": 100000,
+            "fee": 1,
+            "data": None
+        })
+
+        self.assertEqual("00c3c0d1af80b84c3b3de4e3d79d5c8cc950e044098c969953d68bf9cee68d7b53305dbaac7514a06dae935e40d599caf1bd8a243c00000000000186a00000000000000001000af84c01239b16cee089836c2af5c7b1dbb22cdc0b4864349f7f3805909aa8cf24e4c1ff0461832e86f3624778a867d5f2ba318f92918ada7ae28d70d40c4ef1d6413802", result)
+
+    def test_sendTransaction(self):
+        SessionStub.testData = Fixtures.sendTransaction()
+
+        transaction = {
+            "from": "NQ39 NY67 X0F0 UTQE 0YER 4JEU B67L UPP8 G0FM",
+            "fromType": AccountType.BASIC,
+            "to": "NQ16 61ET MB3M 2JG6 TBLK BR0D B6EA X6XQ L91U",
+            "toType": AccountType.BASIC,
+            "value": 1,
+            "fee": 1,
+            "data": None
+        }
+
+        result = self.client.sendTransaction(transaction)
```

Comment:
in the client definition it says "transaction is hex encoded, signed TX" but it's JSON here.

Reply:
Yes, it's the issue with the doc in `sendTransaction`. I'll fix it and double check the other docs.

Reply:
The doc is fixed now.